test/sql/utils/common.sql

```diff
@@ -41,3 +41,20 @@ BEGIN
 END;
 $BODY$
 LANGUAGE plpgsql;
+
+-- Determines if the provided SQL query uses an "Index Scan" by examining its EXPLAIN plan.
+-- Reason: The output of EXPLAIN is not consistent across postgres versions
+CREATE OR REPLACE FUNCTION has_index_scan(query_text text) RETURNS boolean AS $$
+DECLARE
+    plan_row RECORD;
+    found boolean := false;
+BEGIN
+    FOR plan_row IN EXECUTE 'EXPLAIN ' || query_text LOOP
```

Comment:
I think the function would be clearer if EXPLAIN was passed with the query at callsite.
Then, it would be clear this function just inspects the output of the EXPLAIN query.